plugins/scaffolder-common/src/permissions.ts

```diff
@@ -42,6 +49,18 @@ export const actionExecutePermission = createPermission({
   resourceType: RESOURCE_TYPE_SCAFFOLDER_ACTION,
 });
 
+/**
+ * This permission is used to authorize actions that involve access the action registry
+ *
+ * @alpha
+ */
+export const actionReadPermission = createPermission({
```

Comment:
mmm I am not sure about this action, as some of the existing permission rules are bery bound to `actionExecutePermission`. For example `hasProperty`, `hasBooleanProperty`, etc... won't be applicable to this permission, but using this permission together with the `hasActionId` rule might make sense.

Do you have any specific use cases in mind where some adopters would want to completely hide the existence of some actions?

Reply:
Some users using backstage might not be given access to the scaffolder at all, so it won't make sense for them to be able to see the list of installed actions.

Reply:
@vinzscam what do you think?

Reply:
yeah I would skip it for now. This is a bit borderline as the `/v2/actions` endpoint is more used for documentation purposes, which it might be a bit problematic if it was hidden for some users